src/cmr/concepts/collectionDraft.js

```diff
@@ -84,7 +85,7 @@ export default class CollectionDraft extends Concept {
 
     // Construct the promise that will request data from the umm endpoint
     return mmtQuery({
-      conceptType: this.getConceptType(),
+      draftType: 'CollectionDraft',
```

Comment:
Shouldn't this say this.getConceptType() still ?

Reply:
Agree, I changed it back to this.getConceptType(). Originally hard coded them for testing purpose, forgot to change it back.